mariner/util.py

```diff
@@ -0,0 +1,22 @@
+import os
+import pecan
+
+
+def generate_inventory_file(group_name, hosts, task_uuid):
+    """
+    Generates a host file to use with an ansible-playbook call.
+
+    Given a group_name, a lists of hosts and a task UUID a file
+    will be written to disk and a path to the file will be returned.
+    """
+    result = []
+    result.append("[{0}]".format(group_name))
+    if not isinstance(hosts, list):
+        hosts = [hosts]
+    result.extend(hosts)
+    result_str = "\n".join(result)
+    inventory_root = pecan.conf.inventory_root_path
```

Comment:
so this could be an optional argument, to make it easier to test, that way you can just pass in whatever path you want if it needs be